packages/connect-core/src/utils/app.ts

```diff
@@ -1,39 +1,43 @@
 import { utils as ethersUtils } from 'ethers'
-import { AppIntent } from '../types'
```

Comment:
Update a few of the helpers with the latest changes on aragon.js and aligned with the refactoring.